src/Microsoft.AspNetCore/KestrelServerOptionsSetup.cs

```diff
@@ -27,7 +31,7 @@ public void Configure(KestrelServerOptions options)
 
         private void BindConfiguration(KestrelServerOptions options)
         {
-            var certificateLoader = new CertificateLoader(_configurationRoot.GetSection("Certificates"));
+            var certificateLoader = new CertificateLoader(_configurationRoot.GetSection("Certificates"), _loggerFactory);
```

Comment:
@danroth27 I think we should put `CertificateLoader` in DI.

@davidfowl What do you think?